src/mito2/src/engine.rs

```diff
@@ -424,6 +424,21 @@ impl EngineInner {
         receiver.await.context(RecvSnafu)?
     }
 
+    fn get_last_seq_num(&self, region_id: RegionId) -> Result<Option<SequenceNumber>> {
+        // Reading a region doesn't need to go through the region worker thread.
+        let region = self
+            .workers
+            .get_region(region_id)
+            .context(RegionNotFoundSnafu { region_id })?;
+        let version = region.version();
+        let seq = version
+            .memtables
+            .list_memtables()
```

Comment:
Can `memtables` provide a `last_memtable` method? cc @evenyag

Reply:
We should use the committed sequence in the version as the last sequence.